### PR TITLE
Wazo 1585 pjsip global

### PR DESCRIPTION
**Pull request by pc-m**

### CODE REVIEW

File: alembic/versions/28195459f3d1_migrate_sip_general_to_pjsip_global.py [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+
+def _insert_asterisk_file_variable(section_id, key, value):
+    query = (asterisk_file_variable_table
+             .insert()
+             .values(key=key,
+                     value=value,
+                     asterisk_file_section_id=section_id))
+
+    op.get_bind().execute(query)
```
Comment (fblackburn1) [inline alembic/versions/28195459f3d1_migrate_sip_general_to_pjsip_global.py]:
```suggestion
    op.execute(query)
```

**Comment by fblackburn1** [comment]
The options from staticsip are not removed in the migration script. I don't have a real objection, only that maybe we should not remove them from the populate.sql. What do you think?

**Comment by sboily** [comment]
> The options from staticsip are not removed in the migration script. I don't have a real objection, only that maybe we should not remove them from the populate.sql. What do you think?

I'm completely agree!

**Comment by pc-m** [comment]
I agree, I left the sip options in the populate. This will help debug if there some migration that do not get a working configuration. We will just drop the whole table when we are ready

**Pull request merged**